spec/phonegap/cordova.spec.js

```diff
@@ -136,63 +178,63 @@ describe('phonegap.cordova(options, [callback])', function() {
                 });
             });
 
-            it('should trigger the callback without an error', function(done) {
+            it('should trigger the callback with an error', function(done) {
                 phonegap.cordova(options, function(e) {
                     expect(e).toEqual(jasmine.any(Error));
                     expect(e.exitCode).toEqual(1);
                     done();
                 });
-            });
+            }, TIMEOUT);
         });
 
         describe('when not a valid cordova project', function() {
             beforeEach(function() {
                 cordova.util.isCordova.andReturn(false);
                 shell.exec.andCallThrough();
             });
-
-            it('should trigger the callback without an error', function(done) {
+            // ToDo: @carynbear this not testable with current dependency implementation b/c no cordova to call without project.
```

Comment:
This is also a slow test, as we are calling through down to `shell.exec`. We should revisit what this is testing, and how to test it.